ROMFS/px4fmu_common/init.d/rc.interface

```diff
@@ -30,9 +30,9 @@ then
 fi
 
 # USE_IO is set to 'no' for all boards w/o px4io driver or SYS_USE_IO disabled
-if [ $USE_IO = no ]
+if [ $USE_IO = no -a $AUX_BANK2 = none ]
```

Comment:
The comment does not talk about AUX_BANK2, and it's not explained anywhere what it is, and it's not clear from the name either.